src/__init__.py

```diff
@@ -13448,6 +13535,18 @@ def width(self):
 JM_mupdf_show_warnings = 0
 
 
+# ------------------------------------------------------------------------------
+# Image recompression constants
+# ------------------------------------------------------------------------------
+FZ_RECOMPRESS_NEVER = 0
+FZ_RECOMPRESS_SAME = 1
+FZ_RECOMPRESS_LOSSLESS = 2
+FZ_RECOMPRESS_JPEG = 3
+FZ_RECOMPRESS_J2K = 4
+FZ_RECOMPRESS_FAX = 5
+FZ_SUBSAMPLE_AVERAGE = 0
+FZ_SUBSAMPLE_BICUBIC = 1
```

Comment:
Could we use `mupdf.FZ_RECOMPRESS_NEVER` etc here instead of the constants?
E.g. with `from mupdf import FZ_RECOMPRESS_NEVER, FZ_RECOMPRESS_SAME ...` or `FZ_RECOMPRESS_NEVER = mupdf.FZ_RECOMPRESS_NEVER` etc.